docs/dev/development.md

```diff
@@ -44,3 +44,9 @@ To specify your own configuration that overrides the defaults:
 ```
 yarn start -- --config /path/to/config
 ```
+
+## E2E tests
+
+The e2e tests have been Dockerized, which allows them to be run easily anywhere `docker compose` works. To run them on localhost, first create a `.env` file in the `__tests__/e2e`. `docker compose` will alert you as to which variables must be present.
+
+To run the tests, run `docker compose -f docker compose.yml up --abort-on-container-exit` in the `__tests__/e2e/` directory.
```

Comment:
extra space between `docker` and `compose`

Reply:
I have only fixed the file reference in https://github.com/ibi-group/datatools-ui/pull/1032/commits/a2f26cb92bb282ddfbfb8d8b713f3ff20dca5628.